Models/ExploreParams.cs

```diff
@@ -5,16 +5,19 @@ namespace Explorer.Models
 {
     public class ExploreParams
     {
-        [JsonPropertyName("api_key")]
         [Required]
+        [Display(Name = "api_key")]
+        [JsonPropertyName("api_key")]
         public string ApiKey { get; set; }
 
-        [JsonPropertyName("table_name")]
         [Required]
+        [Display(Name = "table_name")]
+        [JsonPropertyName("table_name")]
         public string TableName {get; set; }
 
-        [JsonPropertyName("column_name")]
         [Required]
+        [Display(Name = "column_name")]
+        [JsonPropertyName("column_name")]
```

Comment:
Come to think of it, there is no specific reason why these fields need to be named `api_key`, `table_name`, and `column_name` in the JSON payload that is sent to the service! Instead of bending backwards trying to conform to that arbitrary API we can just as well rename them so the API uses the parameters `ApiKey`, `TableName, and `ColumnName`. That way the original error messages automagically become correct. What do you think about that?

Reply:
This seems to be a good solution, since there is not standard naming convention for the JSON keys (though using the names with underscores as JSON keys felt quite naturally for me).